packages/fxa-payments-server/src/components/PaymentConfirmation/index.tsx

```diff
@@ -103,11 +112,19 @@ export const PaymentConfirmation = ({
           >
             <p>{planPrice}</p>
           </Localized>
-          {last4 && brand && (
+          {last4 && brand && !isPaypalPayment && (
             <Localized id="payment-confirmation-cc-preview" vars={{ last4 }}>
               <p className={`c-card ${brand.toLowerCase()}`}></p>
             </Localized>
           )}
+          {isPaypalPayment && (
+            <div
+              className={`payment-logo ${payment_provider}`}
+              data-testid="paypal-logo"
+            >
+              {payment_provider}
```

Comment:
Should this be 'PayPal' not 'paypal'?

Reply:
I wondered about this also! I used _paypal_ here for consistency since we're using _paypal_ and _stripe_ (instead of _PayPal_ and _Stripe_ ) elsewhere. I don't have a strong opinion about it though.